Loader/SymfonyFixturesLoader.php

```diff
@@ -9,10 +9,14 @@
 namespace Doctrine\Bundle\FixturesBundle\Loader;
 
 use Doctrine\Bundle\FixturesBundle\DependencyInjection\CompilerPass\FixturesCompilerPass;
+use Doctrine\Bundle\FixturesBundle\FixtureGroupInterface;
 use Doctrine\Common\DataFixtures\DependentFixtureInterface;
 use Doctrine\Common\DataFixtures\FixtureInterface;
-use Doctrine\Common\DataFixtures\Loader;
+use ReflectionClass;
+use RuntimeException;
 use Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader;
+use function array_keys;
```

Comment:
What is this? Why do you need use statments for these standard php functions?

Reply:
Strictly speaking, you’d always have to call \array_keys to let PHP know you’re looking for the function from the global namespace. PHP has a fallback for it, but we decided to no longer rely on it for Doctrine. To avoid prepending every function and class from the global namespace, we import them just like we do classes.

The Doctrine Coding Standard coming up in #269 will require these for all functions used in the file.